drivers/ethernet/Kconfig.mcux

```diff
@@ -121,7 +85,6 @@ config PTP_CLOCK_MCUX
 	  Enable MCUX PTP clock support.
 
 if PTP_CLOCK_MCUX
-
```

Comment:
I'm not sure if I commented already or not, but removal of this line seems like a stray change, it rather stay.